test/e2e/cluster_upgrade.go

```diff
@@ -119,83 +243,92 @@ var _ = Describe("Skipped", func() {
 
 		Describe("kube-push", func() {
 			It("of master should maintain responsive services", func() {
+				unsupported := []string{"gke"}
```

Comment:
Yeah... I did it like this because I'm not sure who (correctly) implements kube-push right now—it's a bash "interface" and I was loath to audit all of the scripts. Well, given this test is skipped by default, I'll just turn it on for GCE only right now. (Not yet pushed.)